TCD1304.py

```diff
@@ -0,0 +1,80 @@
+'''
+Requests for the Code:
+the code should have an power Output on the Voltagesource VDD=5V /VAD=5V
+the code should build a Masterclock with a frequency inbetween 0.8 to 4 MHz, typical 2 MHz. The Voltagerange of the Clock 
+shall be 0-4V
+The ICG is a squarewave with a high level of typical 4V over the time of 3694 Elements of the Clock.
+The SH is a squarewave with a high level of typical 4V and indicates the integrationtime. 
+The integrationtime is started with the falling flank of the signal and and lasts till the next falling flank.
+The analog input of the sensor shall be read by the the osilloscope.
+the analog voltage signal shall be saved over the time and be integrated to get a digital signal.
+the algotithims still needs to be definded.
+
+import argparse
+import csv
+import json
+import platform
+import os.path
+import shutil
+import sys
+import time
+from itertools import zip_longest
+from typing import List, Tuple
+
+import numpy as np
+
+'''
+
+
+import time
+import math
+from pslab.instrument.waveform_generator import PWMGenerator
+from pslab.instrument.oscilloscope import Oscilloscope
+from pslab.serial_handler import SerialHandler
+from pslab.instrument.logic_analyzer import LogicAnalyzer
+from pslab.instrument.power_supply import PowerSupply
+
+frequency_master_clock = fmc
+fmc = float (2e6)
+integration_time = intti
+intti = float (1.85e-3) # time in seconds
+
+class tcd1304():

```

Comment:
Shall we make the class name Uppercase?

Reply:
Yes, if going by the pslab-python coding style, the definition should be `class TCD1304:`. Note the lack of parentheses when inheriting directly from `object`.